src/Nethermind/Nethermind.Blockchain/FullPruning/CopyTreeVisitor.cs

```diff
@@ -116,4 +114,9 @@ public void Finish()
             _concurrentWriteBatcher.Dispose();
         }
     }
+
+    public interface ICopyTreeVisitor : IDisposable
```

Comment:
Problem. ITreeVisitor need to have a generic parameter which would cause ICopyTreeVisitor to have a generic parameter which is a problem at FullPruner.

Reply:
A single purpose interface in this case is ok I believe.